contrib/cn.spec

```diff
@@ -0,0 +1,54 @@
+%global source_version 2.0.3
+%global tag 1
+%global provider        github
+%global provider_tld    com
+%global gopath          %{_datadir}/gocode
+
+Name:           cn
+%global project         %{name}
+%global repo            %{name}
+%global provider_prefix %{provider}.%{provider_tld}/%{project}/%{repo}
+%global import_path     %{provider_prefix}
+Version:        %{source_version}
+Release:        %{tag}%{?dist}
+Summary:        A all-in-one tool to bootstrap ceph cluster into a container
```

Comment:
A client tool to bootstrap S3 gateways that leverages container technologies.

Reply:
Fixed